var/spack/repos/builtin/packages/charmpp/package.py

```diff
@@ -48,6 +48,7 @@ class Charmpp(Package):
     # Patch for AOCC
     patch("charm_6.7.1_aocc.patch", when="@6.7.1 %aocc", level=1)
     patch("charm_6.8.2_aocc.patch", when="@6.8.2 %aocc", level=3)
+    patch("charm_6.10_aocc.patch", when="@6.10.0:6.10.2 %aocc")
```

Comment:
Is there any submission upstream for these patches? It would be good if support for a given compiler could be contributed upstream and Spack could work with pristine sources.

Reply:
Agreed.  At the current time we have no traction to get patches upstreamed.  Hopefully with upstream switch to CMake for Charm 7.0 onewards this becomes a non-issue.

Reply:
@amd-toolchain-support If I may, why is there no traction to get the patch upstreamed, considering it is ready and submitted to Spack? Asking generally also in reference to PRs like #35504 and #35583